packages/editor/src/components/rich-text/index.native.js

```diff
@@ -520,7 +520,7 @@ export class RichText extends Component {
 
 			this.lastEventCount = undefined; // force a refresh on the native side
 		}
-		let minHeight = 0;
+		let minHeight = this.props.fontSize ? this.props.fontSize : styles[ 'editor-rich-text' ].minHeight;
```

Comment:
👋 @SergioEstevao , I think the fontSize value is too small as an initial value for the Android side. If you see the minHeights previously used (https://github.com/wordpress-mobile/gutenberg-mobile/blob/develop/src/variables.android.scss#L9-L11) you'll notice that the values are greater than 50. A quick test I did with this current implementation shows a value of `24` in fontSize.

This way, the glitch is still noticeable in the title block for example.